isos/bootstrap/bootstrap

```diff
@@ -19,24 +22,30 @@ drivers=("vmxnet3" \
 	"nf_defrag_ipv4" \
 	"ipt_REJECT"\
 	"nf_reject_ipv4"\
-	"nfsv3")
+	"nfsv3" \
+)
 
 
 for i in ${drivers[@]}; do
-    /usr/sbin/modprobe $i
+    modprobe $i
 done
 
-MOUNTPOINT="/mnt/containerfs"
+# create devices with proper permissions
+udevadm control --reload-rules
+udevadm trigger --type=subsystems --action=add
+udevadm trigger --type=devices --action=add
 
-mkdir -p /mnt/containerfs
+MOUNTPOINT="/mnt/containerfs"
+mkdir -p $MOUNTPOINT
 
 echo "Waiting for rootfs"
 while [ ! -e /dev/disk/by-label/containerfs ]; do sleep 0.1;done
 # https://github.com/vmware/vic/issues/6379
 # grab dmesg output and dump to debug log if mount doesn't occur in a useful timeframe (2min)
-if timeout --signal=KILL 2m mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
+if timeout -s KILL 2m mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     # ensure mountpoint exists
-    mkdir -p ${MOUNTPOINT}/.tether
+    # setup udev directories for switch_root
```

Comment:
These should absolutely already exist in the mount point - they should be part of the scratch image so we do not get a diff. If this was actually _necessary_ rather than precautionary then we need to figure out why.

Reply:
The hello-world image did not container dev.